packages/drivers/odsp-driver-definitions/src/factory.ts

```diff
@@ -7,6 +7,14 @@ export interface ISnapshotOptions {
     blobs?: number;
     deltas?: number;
     channels?: number;
+
+    /*
+     * Maximum time before the cache is expired (in milliseconds)
+     * The default should be 2 * 24 * 60 * 60 * 1000, or 2 days. Leaving this value undefined will
+     * leave cache expiry disabled.
+     */
+    maxSessionActiveTime?: number;
```

Comment:
I believe the purpose of your changes is to put a hard limit on cache duration. I.e. disallow usage of cache longer than 2 days. Exposing it as a policy you do reverse - you allow hosts to put any number they want, thus have not guarantees on actual duration.

Reply:
You should remove this and use a default value for now. Let's figure out the policies when we have the follow up change that adds session expiry timer.